prow/cluster/starter.yaml

```diff
@@ -0,0 +1,247 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: plugins
+data:
+  plugins:
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config
+data:
+  config: |
+    prowjob_namespace: default
+    pod_namespace: default
+    sinker:
+      resync_period: 1h
+      max_prowjob_age: 24h
+      max_pod_age: 12h
+    periodics:
```

Comment:
add a commented out presubmits postusbmits?

Reply:
That will be in the "TODO Add more jobs by modifying config.yaml" section.